public/data/ruchiket.json

```
@@ -0,0 +1,28 @@
+{
+    "name": "Ruchiket",
+    "type": "personal",
+    "bio": "18| Learning Web Developement | Doing #100DaysOfCode Challenge",
+    "avatar": "https://avatars.githubusercontent.com/u/75791209?v=4",
+    "links": [
+        {
+            "name": "Github profile",
+            "url": "https://github.com/Ruchiket100",
+            "icon": "github"
+          },
+      {
+        "name": "Twitter Threads (on Js, React and more)",
+        "url": "https://twitter.com/BorseRuchiket",
+        "icon": "twitter"
+      },
+      {
+        "name": "Instagram",
+        "url": "https://www.instagram.com/ruchikets_world/",
+        "icon": "instagram"
+      },
+      {
+        "name": "Blogs (I write about my dev journey and tech stuff)",
+        "url": "https://hashnode.com/@Ruchiket",
+        "icon": "hashnode"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

This will be fixed by the github action